src/ecs/resources/input.rs

```diff
@@ -333,9 +333,9 @@ impl InputHandler {
     }
 
     /// Returns the value of an axis by the i32 id, if the id doesn't exist this returns None.
-    pub fn axis_value(&self, id: i32) -> Option<f32> {
+    pub fn axis_value(&self, id: &str) -> Option<f32> {
```

Comment:
Since this is a user-facing API, could we use `AsRef<str>` instead of `&str` in all the `axis_` methods so users can easily pass in different kinds of string types?